main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"os"
 	"os/signal"
 	"syscall"
-
+	
```

Comment:
There shouldn't be any diff in main, ideally you could just revert this. I assume you have "keep indents on new lines" on on vscode, but we dont want PRs touching uunecesary files. For now I will approve and merge to main but please make sure to undo any unnecesary diffs in the fututre. @Alishah634

Reply:
Understood thank you!
Will implement this in the future.